bootstrap.go

```diff
@@ -468,9 +468,9 @@ func upgradeToVer12(s Session) {
 	defer terror.Call(r.Close)
 	row, err := r.Next()
 	for err == nil && row != nil {
-		user := row.Data[0].GetString()
-		host := row.Data[1].GetString()
-		pass := row.Data[2].GetString()
+		user, _ := row.GetString(0)
```

Comment:
seems like we have `row.IsNULL` methods, so the boolean `isNull` is not necessary?

Reply:
Yes, I will remove it in future PR.